upgrading/topics/keycloak/changes-18_0_0.adoc

```diff
@@ -18,3 +18,41 @@ the migration. Consider these possible actions:
 insert into CLIENT_SCOPE_CLIENT (CLIENT_ID, SCOPE_ID, DEFAULT_SCOPE) select CLIENT.ID as CLIENT_ID, CLIENT_SCOPE.ID as SCOPE_ID, true as DEFAULT_SCOPE
 from CLIENT_SCOPE, CLIENT where CLIENT_SCOPE.REALM_ID='test' and CLIENT_SCOPE.NAME='acr' and CLIENT.REALM_ID='test' and CLIENT.PROTOCOL='openid-connect';
 ```
+
+= OpenID Connect Logout
+
+Previous versions of {project_name} had support for automatic logout of the user and redirecting to the application by opening logout endpoint URL like
```

Comment:
change "support for automatic logout"
  to active voice
"supported automatic logout"  
change "like" to "such as"